pkgs/applications/virtualization/umoci/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, fetchFromGitHub, buildGoPackage }:
+
+buildGoPackage rec {
+  pname = "umoci";
+  version = "0.4.4";
+
+  goPackagePath = "github.com/openSUSE/umoci";
+
+  src = fetchFromGitHub {
+    owner = "openSUSE";
+    repo = "umoci";
+    rev = "v${version}";
+    sha256 = "1mmk9y6xk0qk5rgysmm7x16b025zzwa2sd13jd32drd48scai2dw";
+  };
+
+  meta = {
```

Comment:
Missing a `maintainers` attr, `description` should not start with the name of the package.

Reply:
description: it's a recursive acronym. Copy pasted from their github repo
maintainers: done